apps/portals/crc-researcher/src/config/handleParticipantWorkflowChange.tsx

```diff
@@ -2,24 +2,17 @@ import {
   PartialRow,
   TableUpdateTransactionRequest,
 } from '@sage-bionetworks/synapse-types'
-import { Row, QueryResultBundle } from '@sage-bionetworks/synapse-types'
-import { SynapseClient } from 'synapse-react-client'
-
-type CustomControlCallbackData = {
```

Comment:
Use exported `CustomControlCallbackData` type to ensure accuracy

Reply:
Note that this is essentially dead code at this point.  We just need a redirector so that the CRC site is unreachable, then we can delete the crc-researcher (researcher facing) config